README.md

```diff
@@ -79,6 +85,27 @@ exports.handler = handler;
 Tasks that use this library are just standard AWS Lambda tasks. Information on
 creating release packages is available [here](https://docs.aws.amazon.com/lambda/latest/dg/deployment-package-v2.html).
 
+## Usage in Cumulus Deployments
+
+During deployment, Cumulus will automatically obtain and inject the [Cumulus Message Adapter](https://github.com/cumulus-nasa/cumulus-message-adapter) zip into the compiled code and create a zip file to be deployed to Lambda.
+
+A task using the message adapter would be configured in lambdas.yml as follows:
+
+```yaml
+NodeTest:
+  handler: index.handler
+  timeout: 300
+  memory: 256
+  source: 'node_modules/@cumulus/task-task/dist/'
+  useMessageAdapter: true
+```
+
+## Development
+
+### Running Tests
+
+To run the tests for this package, run ```npm run test```
```

Comment:
`npm test`.  Also, you can use single backticks here, you don't need three.

Reply:
Done.